doc/man3/X509_get_extension_flags.pod

```diff
@@ -24,6 +25,7 @@ X509_get_proxy_pathlen - retrieve certificate extension data
  uint32_t X509_get_extended_key_usage(X509 *x);
  const ASN1_OCTET_STRING *X509_get0_subject_key_id(X509 *x);
  const ASN1_OCTET_STRING *X509_get0_authority_key_id(X509 *x);
+ const ASN1_OCTET_STRING *X509_CRL_get0_authority_key_id(const X509_CRL *crl);
```

Comment:
Please add this new function to the `HISTORY` section. It will be a new function in 3.5.